Wiki/de/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Gridcoin Wiki
+layout: wiki-de
+---
+
+## Gridcoin Einführung
+
+Gridcoin (Ticker: GRC) ist eine dezentrale, mathematische,
```

Comment:
I would omit the term "mathematische" here.

Reply:
I know my text sounds a bit strange in some sentences, I tried to translate it to the letter, and not to add or leave anything. But I see, it would be better to brush it up a bit. Feel free to make some changes. :)